docs/GettingStartedDocs/Contributors/WindowsManualSGX1FLCDCAPPrereqs.md

```diff
@@ -0,0 +1,80 @@
+# SGX1+ FLC mode: Manually Installing Intel's PSW and Intel Data Center Attestation Primitives (DCAP) [Experimental]
```

Comment:
I'd suggest simplifying this header and drop the outdated [Experimental] label/references to "upcoming support" in the following lines.

> ## Installing Intel Data Center Attestation Primitives (DCAP) and its dependencies on Windows
>
> To use the Intel DCAP libraries on a system supporting SGX with Flexible Launch Control (FLC), you will need to install the following:
> - [Intel Platform Software for Windows (PSW) v2.4](http://registrationcenter-download.intel.com/akdlm/irc_nas/15654/Intel%20SGX%20PSW%20for%20Windows%20v2.4.100.51291.exe)
> - A DCAP client for your data center attestation provider, such as the [Azure DCAP client for Windows](https://github.com/Microsoft/Azure-DCAP-Client/tree/master/src/Windows).
> - [Intel Data Center Attestation Primitives (DCAP) Libraries v1.2](http://registrationcenter-download.intel.com/akdlm/irc_nas/15650/Intel%20SGX%20DCAP%20for%20Windows%20v1.2.100.49925.exe)

Reply:
Fixed